src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -658,6 +660,30 @@ void ApplyLiteralProperties()
 			}
 		}
 
+		/// <summary>
+		/// Override font properties at the end of the app.xaml ctor.
+		/// </summary>
+		/// <param name="writer"></param>
+		private void ApplyFontsOverride(IndentedStringBuilder writer)
+		{
+			var themes = new[] { "Default", "Light", "HighContrast" };
```

Comment:
Can this be stackalloc-ed? Otherwise, it should be a static array to reduce multiple unnecessary allocations

Reply:
This particular invocation is done only once per head; it's not going to change much of the allocation profile. 

Also, unless I'm mistaken stack allocating still requires allocating an array (unless it would have been implemented in this never implemented [proposition](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.3/stackalloc-array-initializers.md)).

Reply:
The proposal you linked should be implemented in C# 7.3.